samples/hello-oboe/src/main/java/com/google/oboe/samples/hellooboe/MainActivity.java

```diff
@@ -65,6 +67,64 @@ public class MainActivity extends Activity {
     private Timer mLatencyUpdater;
     private boolean mScoStarted = false;
 
+    /** Commands for background thread. */
+    private static final int WHAT_START = 100;
+    private static final int WHAT_STOP = 101;
+    private static final int WHAT_SET_DEVICE_ID = 102;
+    private static final int WHAT_SET_AUDIO_API = 103;
+    private static final int WHAT_SET_CHANNEL_COUNT = 104;
+
+    private class MyBackgroundRunner extends BackgroundRunner {
+        int audioApi;
```

Comment:
Should we explicitly define default values for these?

Reply:
Java always initializes variables to zero so we typically do not init them. But I will document that the default zero values correspond to the UNSPECIFIED values for the three parameters.